packages/stark-ui/assets/styles/_typography.scss

```diff
@@ -16,12 +16,58 @@ $typography-config: mat-typography-config(
   $headline: mat-typography-level-create(map-get($typography-theme, headline)),
   $title: mat-typography-level-create(map-get($typography-theme, title)),
   $subheading-2: mat-typography-level-create(map-get($typography-theme, subheading-2)),
-  /*Menu, List, Option*/ $subheading-1: mat-typography-level-create(map-get($typography-theme, subheading-1)),
+  $subheading-1: mat-typography-level-create(map-get($typography-theme, subheading-1)),
   $body-2: mat-typography-level-create(map-get($typography-theme, body-2)),
   $body-1: mat-typography-level-create(map-get($typography-theme, body-1)),
   $caption: mat-typography-level-create(map-get($typography-theme, caption)),
   $button: mat-typography-level-create(map-get($typography-theme, button)),
-  /* Line-height must be unit-less fraction of the font-size.*/ $input: mat-typography-level-create(map-get($typography-theme, input))
+  $input: mat-typography-level-create(map-get($typography-theme, input))
 );
 
 @include mat-core($typography-config);
+
+body {
+  font-family: mat-font-family($typography-config);
+  font-size: mat-font-size($typography-config, body-1);
+  font-weight: mat-font-weight($typography-config, body-1);
+  line-height: mat-line-height($typography-config, body-1);
+}
+
+h1 {
+  font-size: mat-font-size($typography-config, headline);
+  font-weight: mat-font-weight($typography-config, headline);
+  line-height: mat-line-height($typography-config, headline);
+  margin: 0 0 16px;
+}
+
+h2 {
+  font-size: mat-font-size($typography-config, title);
+  font-weight: mat-font-weight($typography-config, title);
+  line-height: mat-line-height($typography-config, title);
+  margin: 0 0 16px;
+}
+
+h3 {
+  font-size: mat-font-size($typography-config, subheading-2);
+  font-weight: mat-font-weight($typography-config, subheading-2);
+  line-height: mat-line-height($typography-config, subheading-2);
+  margin: 0 0 16px;
+}
+
+h4 {
+  font-size: mat-font-size($typography-config, subheading-1);
+  font-weight: mat-font-weight($typography-config, subheading-1);
+  line-height: mat-line-height($typography-config, subheading-1);
+  margin: 0 0 16px;
+}
+
+// Pretty print fix
+.stark-pretty-print {
```

Comment:
I think these rules for the StarkPrettyPrint should be placed in the SCSS of that component. Normally the developer can easily override them...